1.0/docs/devguide/styling.md

```diff
@@ -342,112 +421,195 @@ Example usage of `my-toolbar`:
+    <dom-module id="my-toolbar">
 
       <template>
 
         <style>
           :host {
-            --my-toolbar-color: red;
+            padding: 4px;
+            background-color: gray;
+          }
+          .title {
+            color: var(--my-toolbar-title-color);
           }
         </style>
 
-        <my-toolbar>My awesome app</my-toolbar>
+        <span class="title">{{title}}</span>
+
+      </template>
+
+      <script>
+        Polymer({
+          is: 'my-toolbar',
+          properties: {
+            title: String
+          }
+        });
+      </script>
+
+    </dom-module>
+
+
+    <dom-module id="my-element">
+
+      <template>
+
+        <style>
+          :host {
+            --my-toolbar-title-color: red;
+          }
+        </style>
+
+        <my-toolbar title="My awesome app"></my-toolbar>
         <button on-tap="changeTheme">Change theme</button>
 
       </template>
 
       <script>
         Polymer({
-          is: 'x-custom',
+          is: 'my-element',
           changeTheme: function() {
-            this.customStyle['--my-toolbar-color'] = 'blue';
+            console.log("Currently the style is:", this.getComputedStyleValue('--my-toolbar-title-color') );
+            this.customStyle['--my-toolbar-title-color'] = 'blue';
             this.updateStyles();
+            console.log("After the change, the style is:", this.getComputedStyleValue('--my-toolbar-title-color') );
           }
         });
       </script>
 
     </dom-module>
 
-### Custom properties shim limitations
+As you can see, from this code, `my-element` needs to call `this.updateStyles`,
+or the change in the `--my-toolbar-title-color` custom property won't be carried
+on to `my-toolbar`
 
-Cross-platform support for custom properties is provided in Polymer by a
-JavaScript library that **approximates** the capabilities of the CSS Variables
-specification  *for the specific use case of theming custom elements*, while
-also extending it to add the capability to mixin property sets to rules as
-described above. For performance reasons, {{site.project_title}} **does
-not attempt to replicate all aspects of native custom properties.**
-The shim trades off aspects of the full dynamism possible in CSS in the
-interests of practicality and performance.
+Dynamic effects **are** reflected at the point of a property's application.
 
-Below are current limitations of the shim. Improvements to performance and
-dynamism will continue to be explored.
+For the following example, adding/removing the `urgent` class on the
+`<my-toolbar>` element will have the desired effect, since the dynamism is
+related to *application* of a custom property.
 
-#### Dynamism limitations
+**NOTE: as a selector, I used `:host.urgent span.title` rather than
+**:host(.urgent) span.title mainly because the :host(.selector) syntax hasn't
+**been used at all in the guide before now. I hope it's OK and it's "formally"
```

Comment:
`:host(.urgent)` is correct. `:host.urgent` won't work under shadow DOM (and probably shouldn't work under shady DOM).